relayer/relayer/src/listeners/eth.rs

```diff
@@ -66,120 +66,137 @@ impl EthereumListener {
         loop {
             debug!(target: "EthereumListener", "Ping");
 
-            select! {
+            let unprocessed_block_number = select! {
                 cb_event = circuit_breaker_receiver.recv() => {
                     warn!(target: "EthereumListener","Exiting before handling next block due to a circuit breaker event {cb_event:?}");
                     return Ok(cb_event?);
                 },
+                Ok(unprocessed_block_number) = next_unprocessed_block_number.recv() => {
+                    unprocessed_block_number
+                }
+            };
 
-                Ok (unprocessed_block_number) = next_unprocessed_block_number.recv() => {
-                    // Query for the next unknown finalized block number, if not present we wait for it.
-                    info!(target: "EthereumListener","Waiting for the next finalized block number");
+            // Query for the next unknown finalized block number, if not present we wait for it.
+            info!(target: "EthereumListener", "Waiting for the next finalized block number");
```

Comment:
Since you're already there, can we extract log target? :P

Reply:
Done

Reply:
Yes, log targets are misssing in this repo in general, so does public API rustdocs. We should improve them as we go.